tensorflow/java/src/gen/cc/ops/op_generator.cc

```diff
@@ -0,0 +1,52 @@
+/* Copyright 2017 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Won't the annotations that require it appear in source code? Anyway, let's drop just the `/ops` for now. When I see @kbsriram's PR, I'll get a better idea of the annotation processor.